src/hooks/useQueryParams.ts

```diff
@@ -1,31 +1,39 @@
 import { useCallback } from 'react';
 
 const useQueryParams = () => {
-  const getQueryParam = (queryName) => {
-    const queryParams = new URLSearchParams(window.location.search);
+  const queryParams = new URLSearchParams(window.location.search);
+  const getParsedQueryParam = (queryName) => {
     const queryParam = queryParams.get(queryName);
     if (!queryParam) return [];
 
     return queryParam?.split(',');
   };
 
+  const queryItems = getParsedQueryParam('items');
+  const queryAttributes = getParsedQueryParam('attributes');
+  const quizBasePath = queryParams.get('quizBasePath');
```

Comment:
**issue**: Oh `quizBasePath` is not supplied as a query param here, it's supplied as a prop, see https://github.com/Constructor-io/constructorio-ui-quizzes/pull/145/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR88 you might have to copy some of my changes over though

Reply:
It will most likely be the same domain but just incase customers would like to pre-specify some query params, they can do that 👍 

> Because if we will try to update url with not the same domain we will get the error because of XSS

Ahh good shout, I don't think they'll specify any domain outside of their own 🤔

Reply:
As discussed, let's remove `quizBasePath` for now 👍 sorry for the trouble!